### PR TITLE
the great mono, second edition

### PR DESCRIPTION
**Pull request by bend-n**

### CODE REVIEW

File: src/main/PlagueMain.java [hunk 1]
```diff
@@ -497,12 +499,12 @@ public void init() {
                     return;
                 }
                 PlagueTeam team = teams.get(event.unit.team);
-                team.monos = min((byte) (team.monos + 1), (byte) 255);
+                team.monos = (short) Math.min((int) (team.monos + 1), (int) MONO_LIMIT);
```
Comment (nilederg) [inline src/main/PlagueMain.java:502]: at this point, why not just make mono limit and monos ints

Comment (bend-n) [reply src/main/PlagueMain.java:502]: no

**Review by nilederg — APPROVED** [review]
Yeah sure, lgtm. But it's really unnecessary to use shorts

File: src/main/PlagueMain.java (hunk 1 above)
Comment (nilederg) [reply src/main/PlagueMain.java:502]: fine

**Pull request merged**